tests/integration_tests/monetization_api.test.ts

```diff
@@ -313,8 +313,8 @@
       expect(newInstance.status).to.be.equal("inactive");
       expect(newInstance.pricing).to.be.an('array').with.lengthOf(2);
       expect(newInstance.pricing).to.have.deep.members([{ "eCPM": 10, "Countries": ["US"] }, { "eCPM": 5, "Countries": ["IL"] }])
-      expect(res.rewardedVideo.Vungle).to.be.an('array').with.lengthOf(1);
-      expect(res.rewardedVideo.Vungle[0].rate).to.be.equal(2.5)
+      expect(res.rewardedVideo.liftoffMonetize).to.be.an('array').with.lengthOf(1);
+      expect(res.rewardedVideo.liftoffMonetize[0].rate).to.be.equal(2.5)
```

Comment:
The test is failing because the Vungle object is no longer present in the response. Please confirm if this is an expected change or if the test needs to be updated.